src/infra/alarm/template.yaml

```diff
@@ -455,13 +463,13 @@ Resources:
         - Name: service
           Value: InterventionsProcessorFunction
 
-  DBQueryNoResponseError:
+  DBQueryNoResponseErrorInterventionsProcessor:
```

Comment:
Please make sure this recreates fine. There's cases where alarms need to be deleted before they can be created again, just can't remember the particular scenarios.

Reply:
Hey seems to deploy well 👍 
<img width="943" alt="Screenshot 2024-03-19 at 09 50 59" src="https://github.com/govuk-one-login/account-interventions-service/assets/116737136/8f585796-3a6a-445e-b4c3-46791f8d4981">